sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/GithubAPI.kt

```diff
@@ -1,21 +1,23 @@
 package io.sentry.samples.android
 
 import io.sentry.HttpStatusCodeRange
+import io.sentry.android.okhttp.SentryOkHttpEventListener
 import io.sentry.android.okhttp.SentryOkHttpInterceptor
 import okhttp3.OkHttpClient
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 object GithubAPI {
 
-    private val client = OkHttpClient.Builder().addInterceptor(
-        SentryOkHttpInterceptor(
-            captureFailedRequests = true,
-            failedRequestStatusCodes = listOf(
-                HttpStatusCodeRange(400, 599)
+    private val client = OkHttpClient.Builder().eventListener(SentryOkHttpEventListener())
```

Comment:
`m`: I think we should create a follow-up task to auto-instrument okhttp and the event listener through our gradle plugin, wdyt? Could you create it please?

Reply:
Yeah, I was already thinking about it.
The only thing is that there can only be one eventListener, so we should be careful on how to automatically add it.
Definitely doable, I'll open an issue there

Reply:
hrm, does that mean if someone already has a custom event listener, they won't be able to use ours? is it possible for our intereceptor to accept the other listener as a param to delegate all the calls to it?

Reply:
yeah, we could make it. Should I add it in this pr?

Reply:
I'd do it, yeah